File: backends/php-laravel/app/Domain/Invitation/Condition/InvitationCondition.php
<?php
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
namespace App\Domain\Invitation\Condition;

use App\Support\Repositories\Conditions\AbstractCondition;

/**
 * 招待Conditionクラスです。
 *
 * @author Satoshi Nagashiba <nagashibas@sii-japan.co.jp>
 * @package App\Domain\Invitation\Condition
 */
class InvitationCondition extends AbstractCondition
{
    public ?string $token = null;
}

File: backends/java-spring/src/main/java/com/authorization/domain/invitation/condition/InvitationCondition.java
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
package com.authorization.domain.invitation.condition;

import com.authorization.support.repositories.conditions.AbstractCondition;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * 招待Conditionクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class InvitationCondition extends AbstractCondition {

    private String token;
}
